bot/extensions/safety.py

```diff
@@ -0,0 +1,76 @@
+from discord import Embed
+from discord.colour import Colour
+from discord.ext.commands import Cog
+from discord_slash import cog_ext
+from discord_slash.context import SlashContext
+from discord_slash.utils.manage_commands import create_option, create_choice
+
+from bot import DiscordModeratorWumpus
+from bot.utils import safety_api
+from bot.constants import SAFETY_CATEGORY_NAMES
+
+
+class Safety(Cog):
+    """Commands for fetching information from the Discord Moderator Academy"""
+
+    def __init__(self, bot: DiscordModeratorWumpus) -> None:
+        self.bot = bot
+
+    @cog_ext.cog_subcommand(
+        base="safety",
+        name="category",
+        description="List the safety articles in a particular category.",
+        options=[
+            create_option(
+                name="category",
+                description="The category to be listed.",
+                option_type=4,
+                required=True,
+                choices=[
+                    create_choice(key, value) for (key, value) in SAFETY_CATEGORY_NAMES.items()
+                ],
+            )
+        ],
+    )
+    async def _safety_category(self, ctx: SlashContext, number: int):
+        if number in SAFETY_CATEGORY_NAMES:
+            results = await safety_api.get_api(category=number)
+            embed = Embed(colour=Colour.blurple(), description=f"*{SAFETY_CATEGORY_NAMES[number]}*")
+            embed.set_author(name=f"Discord Safety Portal")
+            for result in results:
+                embed.add_field(
+                    name=result["name"],
+                    value=result["url"],
+                    inline=False,
+                )
+            await ctx.send(embed=embed)
+        else:
+            await ctx.send(content=f"Series {number} could not be found", hidden=True)
```

Comment:
Consider raising a `logger.warning` here - if a user is passing in an invalid parameter somehow (!), it would be good for observability to pick up on this to determine how to alert the core dev team. Logging (should be) a minimal-loss operation as well, so it would add minimal overhead.

Reply:
Added `logger.warning` on L49. *Should* never be called assuming Discord's input validation works, but agreed that it is good to have for logging.